LAUNCH.md

```diff
@@ -0,0 +1,200 @@
+# 🤝 Gflownet Launch tool help
+
+## 💻 Command-line help
+
+```sh
+usage: launch.py [-h] [--help-md] [--job_name JOB_NAME] [--outdir OUTDIR]
+                 [--cpus_per_task CPUS_PER_TASK] [--mem MEM] [--gres GRES]
+                 [--partition PARTITION] [--modules MODULES]
+                 [--conda_env CONDA_ENV] [--venv VENV] [--code_dir CODE_DIR]
+                 [--jobs JOBS] [--dev] [--verbose] [--force]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --help-md             Show an extended help message as markdown. Can be
+                        useful to overwrite LAUNCH.md with `$ python launch.py
+                        --help-md > LAUNCH.md`
+  --job_name JOB_NAME   slurm job name to show in squeue. Defaults to crystal-
+                        gfn
+  --outdir OUTDIR       where to write the slurm .out file. Defaults to
+                        $SCRATCH/crystals/logs/slurm
```

Comment:
Since this is going to `main`, I would change the default output dir to something more generic.